index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <div class="container">
+          <h1 class="logo">My Logo</h1>
```

Comment:
- [ ] As stated in the previous review, Kindly make sure you use the semantic tags for great accessibility to your page. the logo of the page is supposed to be an anchor tag `a` that redirects to the index page as is the convention for UX for a better user experience and for semantic reasons. You may check [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for further Infos.
```suggestion
          <a href="#" class="logo">My Logo</a>
```

Reply:
- [ ] Also make sure only the hamburger menu is grabbed, inside the `nav` tag for semantic reasons since it's the only place we have our navigation links and not even the logo of the page as stated in the previous review

Reply:
hi, @christianbiring1 I have not understood your request. please could you rephrase?